### ISSUE TITLE
CatHack Can`t work on a different ratio screen or another resolutions.

### ISSUE THREAD

**Issue by fSociety-Protected** [time007gc/CatHack#3]
I have a 32:9 monitor 4k, and the CatHack only can scan the left 1080p res of my screen.

**Comment by plumface-gcc** [comment]
Could you post a video/gif of what its doing? 
Do you have multiple monitors? 
Is display scaling set to 100%?

**Comment by fSociety-Protected** [comment]
Ofc....
I gonna record a video.

**Comment by fSociety-Protected** [comment]
https://screenrec.com/share/JfLZS95eK4
No, Im not using more than 1 monitor.
Im playing on a 32:9 monitor, 135cm of screen from left side to right.

**Comment by fSociety-Protected** [comment]
**### The problem is that the program only analyzes the classic 1080p resolution (16:9), but not the 4k (32:9).**

**Comment by plumface-gcc** [comment]
Yep I know why. In the modules/ChampPosition.cs class, it has a hard coded rectangular field of view for the orbwalker.

If you can wait a little bit, I may be able to find a solution by doing some testing on different aspect ratios. Or, you can try compiling by yourself and changing lines 20, 25 and 30 to your desired field of view.

![549df83624592665b05db274bc72dc1d](https://user-images.githubusercontent.com/60156453/140243925-3c83601d-82a6-4cf3-8ac9-03ae1600342c.png)
Rectangle FOV = new Rectangle**_(starting position x, starting position y, width, height)_**

**Comment by fSociety-Protected** [comment]
**I will try to find a solution to my screen resolution/ratio problem with your advice.**

**Hardware wise I'm doing very well, RTX 2080TI, i9-9900K, 64GB RAM, SSD...**

**I'm not worried about a "drop" of FPS, anyway I would be glad if you could help me to solve this problem.**
**If I find a solution before, I will post it as a thread on Github, for people who have this same problem.**

**And maybe, you could make an improvement to your program so that this "problem" won't happen in the future.**
**Thanks for the help.**

**Comment by plumface-gcc** [comment]
I'll do some testing, but I do not have a 4k monitor so it may be a little janky. Add me on discord if you dont mind, we can do some testing there. **Time007#7508**

I'll help you with this tomorrow as its 11pm here already.

**Comment by fSociety-Protected** [comment]
Hey, Im trying to add/send a message in Discord, but Its impossible, pls ADD mee/accept my friend request.
My Discord is: ҉҉#3923
